event.graphqls

```diff
@@ -60,7 +60,7 @@ input EventQueryCondition {
     type: EventType
     time: Duration
     order: Order
-    size: Int
```

Comment:
@wu-sheng @Fine0830 do you think we should keep this in protocol level or remove it?

Reply:
I am good to remove. Wait for @Fine0830 to check what UI prefers, especially about the dashboard part change.

Reply:
Agree to remove it.